src/Composer/Plugin.php

```diff
@@ -255,7 +255,7 @@ protected function isInitialInstall() {
    */
   protected function isNewProject() {
     $composer_json = json_decode(file_get_contents($this->getRepoRoot() . '/composer.json'), TRUE);
-    if (!empty($composer_json['name'] && $composer_json['name'] == 'acquia/blt-project')) {
+    if (isset($composer_json['extra']['blt']['new']) && $composer_json['extra']['blt']['new']) {
```

Comment:
I think we actually need to check the existence of this value with array_key_exists() rather than isset().

Reply:
Merging this would require a corresponding change in `acquia/blt-project`. It may be better to use an OR operator here rather than replacing the condition.